### ISSUE TITLE
GUI should recognise I2CMini, disable current/voltage readouts

### ISSUE THREAD

**Issue by jamesbowman** [jamesbowman/i2cdriver#24]
When device is I2CMini, which has no current or voltage sensing, those readouts should be disabled.

**Comment by williamkapke** [comment]
🤔 How will the I2CMini be identified?

I'm guessing the 'identifier' of the `status response` (`?` command) will say 'I2CMini'?

Since this is a fixed length string- also I guessing it'll be padded at the end with some whitespace.

**Comment by jamesbowman** [comment]
Yes, for I2CDriver the status response field **identifier** field is ``i2cdriver1``:

![image](https://user-images.githubusercontent.com/894763/67114304-7a3e5200-f190-11e9-8cf7-fb45b0e32c29.png)

but for I2CMini has ``i2cdriverm``.
